docs/en/verifying-signatures.wml

```diff
@@ -507,7 +516,7 @@ Primary key fingerprint: EF6E 286D DA85 EA2A 4BA7  DE68 4E2C 6E87 9329 8290
 <!-- Mac OS --><!--
  <div>
    <article class="ac-os">
-     This process does not work on OS X yet due to Apple's codesigning requirement.
+     This process does not work on macOS yet due to Apple's codesigning requirement.
```

Comment:
Please consistently use "OS X", "macOS", or "Mac OS X" throughout the document.

You could choose "macOS", because that's what Apple calls it now. Or you could match the Tor download page.